src/Result/StorePaymentMethodCollection.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace BTCPayServer\Result;
+
+class StorePaymentMethodCollection extends AbstractListResult
+{
+    public function getPaymentMethods(): array
```

Comment:
Not sure if we can use the PHPDoc @return xxx[] here since it;s a mix of 2 types. Maybe look into this or use a common parent type if we have that?

Reply:
would be `@return AbstractStorePaymentMethodResult[]` then? I had it as interface before which would also work but maybe abstract class is more in line with what you started elsewhere in the library.

Reply:
I'm fine either way. What you think is best.

Reply:
done